locales/en/index.yml

```diff
@@ -3182,6 +3182,9 @@ features:
       success:
         title: "Hai aggiunto\nIdentità Digitale!"
         subtitle: Premi Continua per aggiungere {{credentialName}}, tessere o metodi di pagamento al tuo Portafoglio.
+    unsupportedDevice:
+      text: Starting from 02.07.2024 your device may no longer be compatible with IO.
```

Comment:
We could use a parameterized date here, in case we need to use it for something else in the future. What do you think?

Reply:
Sure we can parameterize the date. However, please note that this is a temporary solution. The code will be removed as soon as the necessary period has passed. There will be a more structured solution.

Reply:
I side with @hevelius on this, since this is a temporary workaround that pollutes a screen/component that is going to be removed with the new DS. Informative banner's feature will come in the future and (this) code will be removed after July the 2nd